app/app-start.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+# log something because the python app doesn't log anything on start.
+echo "Starting App."
+
+# TODO - migrate DB
+# Wait for DB to be actually ready, using a probe or healthcheck condition
+# Run the DB migration if any to run.
+# python3 manage.py migrate
```

Comment:
Errrr if this is commented out do we even need the shell script?

Reply:
My intention was to work on the database upgrade bit next once this cleared so I left it in, but I can remove it until that is working if preferred.